src/app/resources/resources.component.ts

```diff
@@ -151,6 +152,24 @@ export class ResourcesComponent implements OnInit, AfterViewInit, OnDestroy {
     this.selection.changed.subscribe(({ source }) => this.onSelectionChange(source.selected));
     this.couchService.checkAuthorization('resources').subscribe((isAuthorized) => this.isAuthorized = isAuthorized);
     this.initialSort = this.route.snapshot.paramMap.get('sort');
+    this.getResources();
```

Comment:
We are already fetching the resources using the resourcesListener so this would be an additional db call